includes/managers/elements.php

```diff
@@ -289,6 +291,17 @@ private function init_categories() {
 			],
 		];
 
+		if ( Plugin::instance()->experiments->is_feature_active( 'favorite-widgets' ) ) {
```

Comment:
```suggestion
		// Not using the `add_category` because it doesn't allow 3rd party to inject a category on top the other.
		if ( Plugin::instance()->experiments->is_feature_active( 'favorite-widgets' ) ) {
```